rj_gameplay/stp/utils/constants.py

```diff
@@ -0,0 +1,49 @@
+# import robocup
+import stp.rc as rc
+import math
+
+DegreesToRadians = math.pi / 180.0
```

Comment:
Is this the style guide for Python constants? I thought it was `DEGREES_TO_RADIANS` (although strictly speaking anywhere you're using this you should just be doing `np.deg2rad/np.rad2deg`).

Reply:
Fixed, left comment for the np thing. @hdemusg please pull this file into your branch when you get the chance.